simon_says/joystick/README.md

```diff
@@ -0,0 +1,26 @@
+# Simon Says joystick package
+This contains a ROS package with a controller node. The controller node can be launched on a device with a connected gamepad and will generate twist messages for a bot's movement and camera.
+
+### Prerequisites
+python inputs module:
+```
+sudo pip install inputs
```

Comment:
rosdep should take care of this, no?

Reply:
https://github.com/ros/rosdistro/pull/18202

Reply:
https://github.com/ros/rosdistro/pull/18202